cspell.json

```diff
@@ -791,6 +793,7 @@
         "authed",
         "authteam",
         "authv",
+        "authzv",
         "Automate's",
         "authn",
         "auxww",
```

Comment:
The minor suggestion related to formatting here, but not related to this PR we can address 


https://github.com/chef/automate/blob/93e2b7586f0a7f6511a8dd9b8fc8901949d5eae4/cspell.json#L863-L866